src/libwalletqt/Wallet.cpp

```diff
@@ -215,6 +215,16 @@ PendingTransaction *Wallet::createTransaction(const QString &dst_addr, const QSt
     return result;
 }
 
+PendingTransaction *Wallet::createTransactionAll(const QString &dst_addr, const QString &payment_id,
+                                                 quint32 mixin_count, PendingTransaction::Priority priority)
+{
+    Bitmonero::PendingTransaction * ptImpl = m_walletImpl->createTransaction(
+                dst_addr.toStdString(), payment_id.toStdString(), boost::optional<uint64_t>(), mixin_count,
```

Comment:
I don't get that point, since src/wallet/api already uses boost in many places. Why does it make a difference ?
And if it does, passing 0 could work, yes. I just thought it a bit iffy and I remembered boost::optional, but if that's a problem I'll use 0. I'd like to understand why this is a problem even though other code there uses boost first though.